src/DotVVM.Framework/Routing/RouteBase.cs

```diff
@@ -184,7 +184,7 @@ public static void AddOrUpdateParameterCollection(IDictionary<string, object> ta
         /// <summary>
         /// Processes the request.
         /// </summary>
-        public abstract IDotvvmPresenter GetPresenter();
+        public abstract IDotvvmPresenter GetPresenter(IServiceProvider provider);
```

Comment:
Btw: The comment is completely misleading.

Reply:
@Leeemons  Please change the comment.